### ISSUE TITLE
`TlsConfig` in Quarkus 3.12 moved to another package

### ISSUE THREAD

**Issue by VijitCoder** [quarkiverse/quarkus-rabbitmq-client#243]
I'm tring to use Quarkus. Started with the newest version - 3.12. And here is the probem with your extension:

> Caused by: jakarta.enterprise.inject.spi.DeploymentException: jakarta.enterprise.inject.UnsatisfiedResolutionException: Unsatisfied dependency for type **io.quarkus.runtime.TlsConfig** and qualifiers [@Default]
	- injection target: parameter 'tlsConfig' of io.quarkiverse.rabbitmqclient.RabbitMQClients constructor
	- declared on CLASS bean [types=[io.quarkiverse.rabbitmqclient.RabbitMQClients, java.lang.Object], qualifiers=[@Default, @Any], target=io.quarkiverse.rabbitmqclient.RabbitMQClients]

It is because `TlsConfig` now in `io.quarkus.tls.runtime.config.TlsConfig` ([proof](https://github.com/quarkusio/quarkus/blob/c43d47e65c03371780fbd5eb8f2cc9666aa56401/extensions/tls-registry/runtime/src/main/java/io/quarkus/tls/runtime/config/TlsConfig.java))

In short:
- 3.11.3 - io.quarkus.runtime.TlsConfig
- 3.12.0 - io.quarkus.tls.runtime.config.TlsConfig

As I see, you have opened PR https://github.com/quarkiverse/quarkus-rabbitmq-client/pull/240 _"Bump quarkus.version from 3.10.0 to 3.12.0"_. I believe there should be more changes than just a framework version.

**Comment by bpasson** [comment]
Thank you for mentioning it. I am aware of this. A new version with the issue resolved will be released shortly.

**Comment by bpasson** [comment]
Version 3.2.0 is released and is compatible with Quarkus 3.12.0. Closing this issue
